test/flows/business_coronavirus_support_finder_flow_test.rb

```diff
@@ -0,0 +1,260 @@
+                    non_domestic_property?: "yes",
+                    sectors?: %w[none]
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      %w[local_1 local_2 national none].each do |response|
+        should "have a next node of results for a #{response} response" do
+          assert_next_node :results, for_response: [response]
+        end
+      end
+
+      should "have a next node of results for a multiple responses" do
+        assert_next_node :results, for_response: %w[local_1 national]
+      end
+    end
+  end
+
+  context "outcome: results" do
+    setup do
+      testing_node :results
+      add_responses business_based?: "england",
+                    business_size?: "0_to_249",
+                    paye_scheme?: "yes",
+                    self_employed?: "yes",
+                    non_domestic_property?: "yes",
+                    sectors?: %w[none],
+                    closed_by_restrictions?: %w[none]
+    end
+
+    should "render job_retention_scheme if paye_scheme? is yes" do
+      assert_rendered_outcome text: "Coronavirus Job Retention Scheme (furlough scheme)"
+    end
+
+    should "render statutory_sick_rebate if paye_scheme? is yes and business_size is 0_to_249" do
+      assert_rendered_outcome text: "Statutory Sick Pay rebate"
+    end
+
+    should "render self_employed_income_scheme if self_employed is yes" do
+      assert_rendered_outcome text: "Self-Employment Income Support Scheme (SEISS)"
+    end
+
+    should "render kickstart_scheme if business_based is not northern_ireland" do
+      assert_rendered_outcome text: "Support to create job placements: Kickstart Scheme"
+    end
+
+    should "render vat_reduction if sectors include retail_hospitality_or_leisure" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
+
+      assert_rendered_outcome text: "VAT reduction for hospitality, accommodation and attractions"
+    end
+
+    should "render traineeships if business_based is england" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
+
+      assert_rendered_outcome text: "Traineeships"
+    end
+
+    should "render apprenticeships if business_based is england" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
+
+      assert_rendered_outcome text: "Apprenticeships"
+    end
+
+    should "render tlevels if business_based is england" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
+
+      assert_rendered_outcome text: "T Levels"
+    end
+
+    should "render council_grants if sector retail_hospitality_or_leisure" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
+
+      assert_rendered_outcome text: "Additional schemes and grants from your local council"
+    end
+
+    should "render additional_restrictions_grant if business_based is england" do
+      add_responses sectors?: %w[retail_hospitality_or_leisure]
```

Comment:
I don't think we need `add_responses sectors?: %w[retail_hospitality_or_leisure]`?